scripts/bootstrap.sh

```diff
@@ -39,6 +39,9 @@ _bootstrap_or_activate() {
 EOF
     )"
 
+    PW_PROJECT_ROOT="$_CHIP_ROOT"
```

Comment:
What's this for?

Reply:
Pigweed's virtualenv setup requires this be correctly set, for e.g. python modules outside of Pigweed repo be possible to install.

Reply:
I find the environment creep problematic, but it's not your fault so I'll take it up elsewhere.